sympy/vector/functions.py

```diff
@@ -124,22 +124,19 @@ def express(expr, system, system2=None, variables=False):
         return expr
 
 
-def directional_derivative(scalar, vect):
+def directional_derivative(direction_vector, field):
```

Comment:
In mathematics they usually write the directional vector first, but in SymPy are we sure we want to do this? The usual convention is to put the independent variable after the derivative, e.g. `Derivative(x**3, x)`, `diff(x**3, x)`.